resources/views/admin/story.blade.php

```diff
@@ -0,0 +1,28 @@
+{{--@extends('layouts.app')--}}
+{{--@section('content')--}}
+@php
+    /**
+     * @var \App\Models\Drawing[] $drawings
+     */
+@endphp
```

Comment:
Ik weet niet zo goed wat dit codeblock nou precies doet en of dit dus code is die we nodig hebben.

Ook aangezien het de pagina niet lijkt te breken op het moment dat ik dit zelf lokaal weg haal.